code/modules/mob/living/simple_animal/hostile/megafauna/onestarboss.dm

```diff
@@ -541,13 +456,13 @@ Effects demand parent code, so ..()
 									step(B, SOUTH)
 									step(A, NORTH)
 									step(B, SOUTH)
-							spawn(80 MILLISECONDS)
+							spawn(600 MILLISECONDS)
```

Comment:
```suggestion
							spawn(6 SECONDS)
```
Our `MILLISECONDS` are ten times as long as actual milliseconds for some reason, so this conversion is correct.
Not sure if you even need that kind of precision, assuming it is even achievable in BYOND.